src/main/scala/com/itechart/ny_accidents/service/metric/DayPeriodMetricService.scala

```diff
@@ -0,0 +1,52 @@
+package com.itechart.ny_accidents.service.metric
+
+import java.util.Calendar
+
+import com.itechart.ny_accidents.constants.GeneralConstants._
+import com.itechart.ny_accidents.entity.MergedData
+import com.itechart.ny_accidents.service.DayPeriodService
+import org.apache.spark.rdd.RDD
+
+object DayPeriodMetricService {
+  private val sunriseSchedule = DayPeriodService.sunrisesSunsets
```

Comment:
no need to copy this variable
just import DayPeriodService.sunriseSunsets

Reply:
done
https://github.com/itechartbigdatalab/NY-Accidents-Rate/pull/22/commits/f3157754a96d572ce6212f6e6d66203e143c6038